demo-java/app/src/main/java/com/example/placesdemo/PlaceIsOpenActivity.java

```diff
@@ -0,0 +1,318 @@
+package com.example.placesdemo;
+
+import android.annotation.SuppressLint;
+import android.app.DatePickerDialog;
+import android.app.TimePickerDialog;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.View;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.AdapterView;
+import android.widget.AdapterView.OnItemSelectedListener;
+import android.widget.ArrayAdapter;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.google.android.gms.tasks.Task;
+import com.google.android.libraries.places.api.Places;
+import com.google.android.libraries.places.api.model.Place;
+import com.google.android.libraries.places.api.model.Place.Field;
+import com.google.android.libraries.places.api.net.FetchPlaceRequest;
+import com.google.android.libraries.places.api.net.FetchPlaceResponse;
+import com.google.android.libraries.places.api.net.IsOpenRequest;
+import com.google.android.libraries.places.api.net.IsOpenResponse;
+import com.google.android.libraries.places.api.net.PlacesClient;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.List;
+import java.util.Locale;
+import java.util.TimeZone;
+
+/**
+ * Activity to demonstrate {@link PlacesClient#isOpen(IsOpenRequest)}.
+ */
+public class PlaceIsOpenActivity extends AppCompatActivity {
+    private final String defaultTimeZone = "America/Los_Angeles";
+    @NonNull
+    private Calendar isOpenCalendar = Calendar.getInstance();
+
+    private EditText editTextIsOpenDate;
+    private EditText editTextIsOpenTime;
+    private TextView textViewResponse;
+    private Spinner spinnerTimeZones;
+    private FieldSelector fieldSelector;
+    private PlacesClient placesClient;
+    private Place place;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.place_is_open_activity);
+
+        // Retrieve a PlacesClient (previously initialized - see MainActivity)
+        placesClient = Places.createClient(/* context= */ this);
+
+        textViewResponse = findViewById(R.id.textView_response);
```

Comment:
What about using view binding here? We are using it on other demos(see `PlaceDetailsAndPhotosActivity`), and removes a lot of boilerplate code.